config/application.yml.example

```diff
@@ -70,6 +71,7 @@ production:
   otp_delivery_blocklist_findtime: '5'
   otp_delivery_blocklist_maxretry: '5'
   participate_in_dap: 'no'
+  password_pepper: 'rake secret'
```

Comment:
does this mean we should be generating a secret for prod and storing it somehow? How are we dealing with sensitive keys in this deployment setup?